src/components/MediaList/MediaList.tsx

```diff
@@ -38,81 +46,51 @@ interface MediaSourceProps {
   sourceType?: string;
 }
 
-const MediaLicenseContainer = styled.div`
-  padding-bottom: ${spacing.normal};
-`;
-
-const BodyTitle = styled(Text)`
-  font-weight: ${fonts.weight.bold};
-  padding-bottom: ${spacing.xsmall};
-`;
-
 export const MediaListLicense = ({ licenseType, title, sourceTitle, sourceType }: MediaSourceProps) => {
   const { i18n, t } = useTranslation();
   const license = getLicenseByAbbreviation(licenseType, i18n.language);
   const { description } = getLicenseRightByAbbreviation(license.rights[0] ?? "", i18n.language);
 
   const licenseRightsText = license.rights[0] === COPYRIGHTED ? "restrictedUseText" : "licenseText";
-
   return (
-    <MediaLicenseContainer>
-      {title ? (
-        <BodyTitle element="label" margin="none" textStyle="meta-text-medium">
-          {`${title} "${sourceTitle}"`}
-        </BodyTitle>
-      ) : null}
-      <br />
-      <span>
-        {`${t(`license.${sourceType}.${licenseRightsText}`)} `}
-        <LicenseLink license={license} />
-        {`. ${description}`}
-      </span>
+    <div>
```

Comment:
Trengs det ingen styling her? Isåfall, kan dette være en fragment?

Reply:
Jeg vil tro nei, siden dette ligger inne i en flex-container